docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx

```diff
@@ -167,65 +92,60 @@ over the planned route as can be seen in the following figure.
   </figure>
 </center>
 
-### Goal Panel
+### Waypoint Panel
 
 <center>
   <figure>
-    <img src="/img/outdoornav_images/ui_goal_panel.png" />
-    <figcaption>Goal Panel</figcaption>
+    <img src="/img/outdoornav_images/ui_waypoint_panel.png" />
+    <figcaption>Waypoint Panel</figcaption>
   </figure>
 </center>
 
-Enable the "Goal Panel" toggle to open the list of available Goals
+Enable the "Waypoint Panel" toggle to open the list of available Waypoints
 within the selected Mission as shown in the figure above. The user can
-now enable/disable Goals, rearrange the list, rename Goals, add Tasks to
-the end of Goals, and modify the final heading and/or tolerance of each
-Goal.
-
-When the Mission is started, the Goals are executed in the order that
-they appear in the Goals panel (provided that they are enabled).
+now rearrange the list, rename Waypoints, add Tasks to the Waypoints, 
+and modify the final heading and/or tolerance of each Waypoint.
 
-### Rearrange List of Goals
+### Rearrange List of Waypoints
 
-Goals can be rearranged once they have been completed. To do this,
-enable the "Goals Panel" toggle to access the list of Goals. Here, the
-user will be able to drag and drop the Goals to reorder them.
+Waypoints can be rearranged in order of operation in the list. To do this,
+enable the "Waypoints Panel" toggle to access the list of Waypoints. Here, the
+user will be able to drag and drop the Waypoints to reorder them.
 
-### Rename Goal
+### Rename Waypoint
 
-By default, once Goals are created they are assigned a default name
-which is the word "Goal" followed by a numeric value representing the
-the number of Goals that have been created plus one. The user has the
-option to rename these Goals in order for them to have more descriptive
-meaning. The Goal name will appear in the feedback bar once the Mission
-is sent and will let the user know which Goal is currently being
-executed.
+By default, once Waypoints are created they are assigned a default name
+which is the word "Waypoint" followed by a numeric value representing the
+the number of Waypoints that have been created plus one. The user has the
+option to rename these Waypoints in order for them to have more descriptive
+meaning. 
 
-To rename a Goal follow these steps:
+To rename a Waypoint follow these steps:
 
-1.  Enable the "Goal Panel" toggle. If you cannot find this button,
-    see [Goal Panel](#goal-panel) for reference.
-2.  Click the name of the Goal which the user wants to rename.
+1.  Enable the "Waypoint Panel" toggle. If you cannot find this button,
+    see [Waypoint Panel](#waypoint-panel) for reference.
+2.  Click the name of the Waypoint which the user wants to rename.

```

Comment:
We flip between the second person singlular ("you") and third person singular ("the user"). We should choose one or the other and be consistent throughout. Applies to other locations in this file as well.